### PR TITLE
Updates to Paid Time off Page

### PR DESCRIPTION
**Pull request by yomiadetutu1**

### CODE REVIEW

**Review by lynn915 — CHANGES_REQUESTED** [review]
Line 21: 
Communicate your Planned Time Off (PTO) - or PTO. Currently says Paid Time off. 
Line 22: 
Remove "if the public holiday is already listed....". 
Line 23:
This should only apply to FT Staff outside of entity and EOR locations. Their public holidays aren't listed in Rippling so only they should (and can) request this off as "public holiday". 

RE: public holidays listed in the handbook, maybe we should leave those in this year and remove for 2025?

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 1]
```diff
@@ -18,27 +18,27 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
-4. PTO requests must be "approved" in BambooHR to show up in the "Who's Out" calendar. Managers are responsible for approving their direct reports' time off requests in BambooHR.  
-5. As long as you have entered your time off in BambooHR, and you have arranged for coverage while you are gone, you are able to take your vacation before it has been approved in BambooHR. More than 14 consecutive days off require manager approval in BambooHR prior to the vacation starting.
+1. Communicate your Paid Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, bereavement, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
1. Communicate your Planned Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, bereavement, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 2]
```diff
@@ -18,27 +18,27 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
+2. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Sweden, US, UK) are automatically listed in Rippling. If the public holiday is already listed in Rippling, no action is required.
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
2. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Sweden, US, UK) are automatically listed in Rippling. 
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 3]
```diff
@@ -18,27 +18,27 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
+1. Communicate your Planned Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, bereavement, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
+2. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Sweden, US, UK) are automatically listed in Rippling. 
+*For all FT Staff, if there are any statutory public holidays in your region of residence that are not listed in Rippling, please ensure to enter the day in **Rippling > Time Off > Public Holiday** to distinguish it from vacation time.*
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
*For all FT Staff outside of entity and EOR locations, if there are any statutory public holidays in your region of residence that are not listed in Rippling, please ensure to enter the day in **Rippling > Time Off > Public Holiday** to distinguish it from vacation time.* 
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 4]
```diff
@@ -49,44 +49,4 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
-* Day of German Unity \(Germany\) \(10/03/2024\)
-* Labour Day \(Australia\) \(10/07/2024\)
-* Thanksgiving \(Canada\) \(10/14/2024\)
-* Remembrance Day \(Canada: AB, BC, PEI\) \(11/11/2024\)
-* Thanksgiving \(US\) \(11/28/2024\)
-* Christmas Day \(All\) \(12/25/2024\)
-* Boxing Day \(Australia, Canada, Germany, Sweden, UK\)\(12/26/2024\)
-
-It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if a large number of staff will not be attending the meeting\).
+
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md:53]:
```suggestion
### Typical public holidays in our entities and EOR locations- Australia, Canada, Germany, Sweden, UK, and US

We have entities and EOR locations in Australia, Canada, Germany, Sweden, UK, and US, so below are holidays we’re expecting people from those countries to take off. Please see [Communicating Time Off](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#communicating-time-off)*

**Entity and EOR locations holidays for 2024**

* New Year’s Day \(All\) \(1/01/2024\)
* MLK Jr. Day \(US\) \(1/15/2024\)
* Australia Day \(Australia\) \(1/26/2024\)
* Family Day/Islander Day \(Canada\) \(2/19/2024\)
* Good Friday \(Australia Canada, Germany, Sweden, UK\) \(3/29/2024\)
* Easter Monday \(Australia, Germany, Sweden, UK\) \(4/01/2024\)
* Anzac Day \(Australia\) \(4/25/2024\)
* Labour Day \(Germany, Sweden\) \(5/01/2024\)
* Early May Bank Holiday \(UK\) \(5/06/2024\)
* Ascension Day \(Germany, Sweden\) \(5/09/2024\)
* Whit Monday \(Germany\) \(5/20/2024\)
* Victoria Day \(Canada\) \(5/20/2024\)
* Memorial Day \(US\) \(5/27/2024\)
* Spring Bank Holiday \(UK\) \(5/27/2024\)
* National Day \(Sweden\) \(6/06/2024\)
* Midsummer Day \(Sweden\) \(6/24/2024\)
* Canada Day \(Canada\) \(7/01/2024\)
* Independence Day \(US\) \(7/04/2024\)
* Civic Holiday/Provincial Day \(Canada: AB, BC, ON\) \(8/05/2024\)
* Summer Bank Holiday \(UK\) \(8/26/2024\)
* Labor Day \(US, Canada\) \(9/02/2024\)
* Truth and Reconciliation Day \(Canada: BC, PEI\) \(9/30/2024\)
* Day of German Unity \(Germany\) \(10/03/2024\)
* Labour Day \(Australia\) \(10/07/2024\)
* Thanksgiving \(Canada\) \(10/14/2024\)
* Remembrance Day \(Canada: AB, BC, PEI\) \(11/11/2024\)
* Thanksgiving \(US\) \(11/28/2024\)
* Christmas Day \(All\) \(12/25/2024\)
* Boxing Day \(Australia, Canada, Germany, Sweden, UK\)\(12/26/2024\)

It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if a large number of staff will not be attending the meeting\).
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 5]
```diff
@@ -49,10 +49,6 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
 
 Please take off holidays relevant to your culture, resident [country/region](../../../../../company/about-mattermost/list-of-terms.md#country-region), and preferences. When doing so, please follow the time off process in [Communicating Time Off](./#communicating-time-off).
 
-## Family and Friends days
-
-Family and Friends days are automatically added to BambooHR and you don't need to book that time off.
-
 ### Typical public holidays in our entities and EOR locations- Australia, Canada, Germany, Sweden, UK, and US
 
 We have entities and EOR locations in Australia, Canada, Germany, Sweden, UK, and US, so below are holidays we’re expecting people from those countries to take off. Please see [Communicating Time Off](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#communicating-time-off)*
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md:54]:
```suggestion
We have entities and EOR locations in Australia, Canada, Germany, Sweden, UK, and US, so below are holidays we’re expecting people from those countries to take off. Please see [Communicating Time Off](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#communicating-time-off)*

**Entity and EOR locations holidays for 2024**

* New Year’s Day (All) (1/01/2024)

* MLK Jr. Day (US) (1/15/2024)

* Australia Day (Australia) (1/26/2024)

* Family Day/Islander Day (Canada) (2/19/2024)

* Good Friday (Australia Canada, Germany, Sweden, UK) (3/29/2024)

* Easter Monday (Australia, Germany, Sweden, UK) (4/01/2024)

* Anzac Day (Australia) (4/25/2024)

* Labour Day (Germany, Sweden) (5/01/2024)

* Early May Bank Holiday (UK) (5/06/2024)

* Ascension Day (Germany, Sweden) (5/09/2024)

* Whit Monday (Germany) (5/20/2024)

* Victoria Day (Canada) (5/20/2024)

* Memorial Day (US) (5/27/2024)

* Spring Bank Holiday (UK) (5/27/2024)

* National Day (Sweden) (6/06/2024)

* Midsummer Day (Sweden) (6/24/2024)

* Canada Day (Canada) (7/01/2024)

* Independence Day (US) (7/04/2024)

* Civic Holiday/Provincial Day (Canada: AB, BC, ON) (8/05/2024)

* Summer Bank Holiday (UK) (8/26/2024)

* Labor Day (US, Canada) (9/02/2024)

* Truth and Reconciliation Day (Canada: BC, PEI) (9/30/2024)

* Day of German Unity (Germany) (10/03/2024)

* Labour Day (Australia) (10/07/2024)

* Thanksgiving (Canada) (10/14/2024)

* Remembrance Day (Canada: AB, BC, PEI) (11/11/2024)

* Thanksgiving (US) (11/28/2024)

* Christmas Day (All) (12/25/2024)

* Boxing Day (Australia, Canada, Germany, Sweden, UK)(12/26/2024)

It’s up to meeting chairs to decide whether to reschedule if few people can attend (e.g. if a large number of staff will not be attending the meeting).
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 6]
```diff
@@ -49,16 +49,74 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
+
+* Christmas Day (All) (12/25/2024)
+
+* Boxing Day (Australia, Canada, Germany, Sweden, UK)(12/26/2024)
+
+It’s up to meeting chairs to decide whether to reschedule if few people can attend (e.g. if a large number of staff will not be attending the meeting).
+
+**Entity and EOR locations holidays for 2024**
+
 * New Year’s Day \(All\) \(1/01/2024\)
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md:120]:
```suggestion

```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 7]
```diff
@@ -49,16 +49,74 @@ Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or
...
 * MLK Jr. Day \(US\) \(1/15/2024\)
 * Australia Day \(Australia\) \(1/26/2024\)
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion

```

**Pull request closed without merging**